fs/operations.go

```diff
@@ -1583,7 +1583,7 @@ func Cat(f Fs, w io.Writer, offset, count int64) error {
 }
 
 // Rcat reads data from the Reader until EOF and uploads it to a file on remote
-func Rcat(fdst Fs, dstFileName string, in0 io.ReadCloser, modTime time.Time) (err error) {
```

Comment:
Why did you change the API here? returning the object created is useful isn't it?

Reply:
Hm, I'm not quite sure what you're referring to here. I'm actually extending the API to *also* return the `fs.Object` if possible. This was needed so that I can still attach it to the write handle in FUSE – previously `Rcat` would just return `error`.

Reply:
Yes you are right - diff reading failure ;-) Sorry!